File: Safari-609.1.11.1/LayoutTests/http/tests/security/xssAuditor/resources/tag-with-pause.php
<?
header("Content-Type: text/html; charset=utf-8");
?>
<!-- <?php
# Spam a bunch of As to make sure we blow past any buffers.
print str_repeat("A", 2048);
?> -->
<body>
<?php
print "<a ona";
print str_repeat("a", 2000);

flush();
usleep(200000);

print "click=alert(1) ttt>";
?>
Done.

File: LayoutTests/http/tests/security/xssAuditor/resources/tag-with-pause.py
#!/usr/bin/env python3

import sys
import time

sys.stdout.write(
    'Content-Type: text/html; charset=utf-8\r\n\r\n'
    '{}<body>\n'
    '<a ona{}\n'
    '{}'.format('A'*2048, 'a'*2000)
)

sys.stdout.flush()
time.sleep(0.2)

sys.stdout.write(
    'click=alert(1) ttt>\n'
    'Done.\n'
)